plugins/package.json

```diff
@@ -5,6 +5,7 @@
   "description": "A sample Apache Cordova application that responds to the deviceready event.",
   "main": "index.js",
```

Comment:
I think we can/should perhaps declare the "types" field to point at index.d.ts for the module?

Maybe it's automatically picked up based on some convention, but if in a non-standard location I think we have to declare it.

Reply:
yeah,  "main" is quite certainly incorrect. Im not sure if my change is 100% right, in the old SDK we used to copy some of the types into root folder for some reason.

Maybe can verify types are found when typescriptify the app

Reply:
TS type declarations should also help (if they are found) in non-TS apps the are edited in a TS def aware editor like VSCode. They typing info should be found in the JS app & help w/ arg hints etc. if it's working properly I think.

Reply:
Pick this up as follow-up.